src/parse.h

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+using namespace std;
```

Comment:
[same as above] Please don't import the entire std:: namespace. Prefix identifiers with std:: to make it explicit where they're coming from, or import only the part of the namespace you need (if possible).